apps/mobile/src/screens/auth/components/sign-in-form/sign-in-form.tsx

```diff
@@ -1,32 +1,61 @@
 import React from "react";
 
-import {
-	BackgroundWrapper,
-	Button,
-	Link,
-	Text,
-	View,
-} from "~/libs/components/components";
+import { Button, Input, Link, Text, View } from "~/libs/components/components";
 import { RootScreenName } from "~/libs/enums/enums";
+import { useAppForm, useCallback } from "~/libs/hooks/hooks";
+import { globalStyles } from "~/libs/styles/styles";
+import { userSignInValidationSchema } from "~/packages/users/libs/validation-schemas/validation-schemas";
+import { type UserSignInRequestDto } from "~/packages/users/users";
+import { USER_SIGN_IN_DEFAULT_VALUES } from "~/screens/auth/components/sign-in-form/libs/constants";
 
 type Properties = {
-	onSubmit: () => void;
+	onSubmit: (payload: UserSignInRequestDto) => void;
 };
 
-const SignInForm: React.FC<Properties> = () => {
+const SignInForm: React.FC<Properties> = ({ onSubmit }) => {
+	const { control, errors, handleSubmit } = useAppForm<UserSignInRequestDto>({
+		defaultValues: USER_SIGN_IN_DEFAULT_VALUES,
+		validationSchema: userSignInValidationSchema,
+	});
+
+	const handleFormSubmit = useCallback((): void => {
+		void handleSubmit(onSubmit)();
+	}, [handleSubmit, onSubmit]);
+
 	return (
-		<BackgroundWrapper>
-			<View>
-				<Text>Sign In</Text>
-				<Button
-					label="Sign in"
-					onPress={(): void => {
-						// TODO: handle Sign in
-					}}
-				/>
-				<Link label="Go to Sign Up" to={`/${RootScreenName.SIGN_UP}`} />
+		<>
+			<Text preset="heading" size="xl">
+				SIGN IN
+			</Text>
+			<Text style={[globalStyles.mb16]}>
+				No account? Go to{" "}
+				<Link label="Create an account" to={`/${RootScreenName.SIGN_UP}`} />
+			</Text>
+			<Input
+				control={control}
+				errors={errors}
+				label="Email"
+				name="email"
+				placeholder="name@gmail.com"
+			/>
+			<Input
+				control={control}
+				errors={errors}
+				label="Password"
+				name="password"
+				placeholder="********"
```

Comment:
I think we need `secureTextEntry` for password input

Reply:
add secureTextEntry for password like props